app/models/concerns/patient_searchable.rb

```diff
@@ -2,11 +2,12 @@
 module PatientSearchable
   extend ActiveSupport::Concern
 
-  SEARCH_LIMIT = 15
+  DEFAULT_SEARCH_LIMIT = 15
 
   class_methods do
     # Case insensitive and phone number format agnostic!
-    def search(name_or_phone_str, lines = LINES)
+    # pass `search_limit = nil` for no limit.
+    def search(name_or_phone_str, lines: LINES, search_limit: DEFAULT_SEARCH_LIMIT)
```

Comment:
This swaps these to keyword args rather than positional args, right? I'm good with that, though reading the comment does it mean we want to do `search_limit: nil`?

Reply:
oops, yup!